pkg/watcher/skr_webhook_manifest_manager.go

```diff
@@ -18,6 +20,11 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	// always true since unsecured watcher setup will no longer be supported.
+	tlsEnabled = "true"
```

Comment:
I would remove this flag since its not needed

Reply:
Moved it!